### PR TITLE
update graphql and dependencies to support 14.0.2

### PR DESCRIPTION
**Pull request by JustFly1984**
I'm migrating my project to gatsby, and we are heavely depending on graphql@14.0.2
I've updated all dependencies, which causing npm warnings while migrating to gatsby.

Please review and merge.

### CODE REVIEW

**Comment by pieh** [comment]
Please revert any formatting changes (and prettier update) - it makes it really hard to properly review

File: packages/babel-plugin-remove-graphql-queries/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "license": "MIT",
   "main": "index.js",
   "peerDependencies": {
-    "graphql": "^0.13.2"
+    "graphql": "^0.14.2"
```
Comment (sibiraj-s) [inline packages/babel-plugin-remove-graphql-queries/package.json]: this should be `14.0.2` rite?

File: docs/blog/2018-06-14-escalade-sports-from-5000-to-5-in-hosting/index.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Working with another developer, a photographer, and a graphic designer, he found
 
 Another unexpected benefit was the cost reduction of hosting static content, instead of running PHP servers. After migrating to Gatsby, Rose moved the cajunbowfishing.com website hosting over to [Netlify](www.netlify.com).
 
-“Our bill went from $5,000 a month to $5,” he says.
```
Comment (shannonbux) [inline docs/blog/2018-06-14-escalade-sports-from-5000-to-5-in-hosting/index.md]: Is this an error caused by linting?

**Comment by m-allanson** [comment]
Thanks for creating this @JustFly1984 👍 Now that there's an updated version of relay-compiler available, your work has been continued over in #11377 which we're aiming to get merged fairly soon. I'm going to close this PR in favour of the updated one. Thanks again!

**Pull request closed without merging**